herbert_core/admin/@opt_config_manager/opt_config_manager.m

```diff
@@ -131,25 +137,30 @@
             config = obj.current_config_;
         end
 
+        function types = get.pc_types(obj)
+            types = obj.pc_types_;
+        end
+
         function obj = set.this_pc_type(obj,val)
             % explicitly setting pc type for testing or debugging purposes.
             %
             % the type can be set by name (from the list of the names
             % specified in the list definition) or by the number of the pc
             % type in the same list
             if isnumeric(val)
-                n_types = numel(obj.known_pc_types_);
+                n_types = numel(obj.pc_types_);
                 if val>0 && val<=n_types
-                    pc_type = obj.known_pc_types_{val};
+                    val = obj.pc_types_{val};
+                    pc_type = obj.known_pc_types_(val);
```

Comment:
```suggestion
                    pc_type = obj.known_pc_types(val);
```

Reply:
well, using private property within class method is a good practice. No need to go through public interface which is often slower.